base/bucket_n1ql_test.go

```diff
@@ -172,11 +162,7 @@ func TestN1qlFilterExpression(t *testing.T) {
 
 	// Defer index teardown
 	defer func() {
-		// Drop the index
-		err = n1qlStore.DropIndex(ctx, "testIndex_filtered_value")
-		if err != nil {
-			t.Fatalf("Error dropping index: %s", err)
-		}
+		assert.NoError(t, DropAllIndexes(ctx, n1qlStore))
```

Comment:
Why do we need to defer drop all index here when it looks to be done as part of test cleanup in get `getN1QLStore`? Asking so i better understand the test

Reply:
This was just test cleanup I missed.